### PR TITLE
Implement GH-372: Add `/info/cards/` route

### PR DESCRIPTION
**Pull request by mewtaylor**
Also implements a method for handling translation of static files as described in https://github.com/LLK/scratch-www/issues/372#issuecomment-193350144. This also adds links as requested in #362, however we need to add thumbnails to our cdn in scratchr2 as well for that to work properly.

### CODE REVIEW

File: src/views/cards/cards.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+@import "../../colors";
+@import "../../typography";
+
+.intro {
```
Comment (rschamp) [inline src/views/cards/cards.scss]: I think we already have an element with this class name — could you wrap all of the styles for the component in a class so there can't be any interaction? I know practically there won't be, but just for consistency.  E.g., add a `cards` class to the `div.inner`, and then wrap all these styles in `.cards`?

**Comment by rschamp** [comment]
Looks good to me other than one nitpick.

File: src/views/cards/cards.scss (hunk 1 above)
Comment (mewtaylor) [inline src/views/cards/cards.scss]: :+1:

**Pull request merged**